docs/source/installation.rst

```diff
@@ -90,19 +36,22 @@ entry
 
 .. code-block:: yaml
 
-   sqlalchemy.url : postgresql://reg_writer@data-registry-dev-loadbalancer.jrb-test.development.svc.spin.nersc.org:5432/desc_data_registry
+   sqlalchemy.url : postgresql://<username>@dataregistry-release-test-loadbalancer.mcalpine-test.development.svc.spin.nersc.org:5432/desc_data_registry 
+
+where ``<username>`` should either be ``reg_writer`` or ``reg_reader``,
+depending on what account you have access to.
```

Comment:
It would be good to add a note here (or somewhere) something like this:
**Note:**  ``reg_writer`` cannot write to production (schema ``lsst_desc_production``).  There is another more privileged account for that.  If you need to register production entries, please consult one of the dataregistry admins.  Both ``reg_reader`` and ``reg_writer`` _can_ write to the tutorial version of production (schema ``tutorial_production``).